mbed_targets/mbed_tools/cli.py

```diff
@@ -0,0 +1,5 @@
+from mbed_targets.mbed_tools.list_all import list_all as cli
```

Comment:
I'm guess this will change when we want to add more than one command to the API? Seems a bit strange that `list_all == cli` but if it's just the example I think I get it.

Reply:
It is just an example. :)